scvi/utils/_attrdict.py

```diff
@@ -1,40 +1,11 @@
-from copy import deepcopy
+from ml_collections.config_dict import FrozenConfigDict
 
 
-class attrdict(dict):
+class attrdict(FrozenConfigDict):
     """
-    A dictionary that allows for attribute notation (e.g. d.element).
-
-    Parameters
-    ----------
-    recursive
-        If True, recursively converts nested dictionaries into :class:`~scvi.utils.attrdict` objects.
-
-    Notes
-    -----
-    Based off of https://stackoverflow.com/questions/38034377/object-like-attribute-access-for-nested-dictionary.
+    Dummy class to allow for backwards compatibility with old attrdict
+    Now uses ml-collections FrozenConfigDict.
```

Comment:
we should be able to add ml colletions to intersphinx in conf.py and do a proper :class: link to the object.

https://ml-collections.readthedocs.io/en/latest/